### ISSUE TITLE
PoseGroup childrens' children enter poses don't animate

### ISSUE THREAD

**Issue by lettertwo** [Popmotion/popmotion#285]
A posed component's posed children don't seem to animate their 'enter' pose when part of a PoseGroup tree.

or, in pseudocode:
```jsx
const PosedParent = posed.div({enter: {...}, exit: {...}});
const PosedChild = posed.div({enter: {...}, exit: {...}});

render(
  <PoseGroup>
    <PosedParent>
      <PosedChild>I don't seem to 'enter', but i do 'exit'!</PosedChild>
    </PosedParent>
  </PoseGroup>
);

```

Here's a simple pen to illustrate (check the console output): https://codepen.io/lettertwo/pen/xWaVNK?editors=1111

I'm really excited about react-pose and trying real hard to use it in a current project. If you think there's a bug here that i might be able to look into, let me know!

**Comment by anthonysapp** [comment]
You might want to try setting the "animateOnMount" property of the PoseGroup to true.

**Comment by lettertwo** [comment]
The pseudocode above was just meant to convey the idea of the pose tree i'm trying to build. The linked codepen does use `animateOnMount`. Sorry this was confusing! 😖

**Comment by anthonysapp** [comment]
My bad for not checking the pen :). Not sure about the answer here, sorry!

**Comment by lettertwo** [comment]
> I think the idea of the PoseGroup is only to animate direct children as they enter / exit the component tree. Not 100% sure on that, though.

Yeah, that's my understanding, too, but I'm hoping/expecting here that the inferred parent-child relationship that nested Posed components create (documented [here](https://popmotion.io/pose/learn/animating-children/#animating-children-children-react)) will pass the PoseGroup-triggered poses down the tree, and indeed, it seems to work as i'm expecting for the `exit` pose, but not for `enter`.

**Comment by InventingWithMonster** [comment]
This is indeed the intended behaviour - I’ll take a look at it!

**Comment by ConneXNL** [comment]
I also tried to nest two nest groups. For example in a spreadsheet like component with animating columns and rows.

I ran into problems as soon as I tried to use a a <PoseGroup> inside a <PoseGroup> where animating leaves ghost elements.

**Comment by InventingWithMonster** [comment]
@lettertwo here's a quick fix that replicates what `PoseGroup` is doing to `Parent`:

```javascript
<Child initialPose="exit" pose="enter" />
```

Not sure how to fix this right now but looking into it today

**Comment by InventingWithMonster** [comment]
Fixed in 1.1.3

**Comment by InventingWithMonster** [comment]
You'll have to update the pen to use React 16.3.1
